### ISSUE TITLE
alpn examples are inconsistent with draft-schwartz-svcb-dns

### ISSUE THREAD

**Issue by mnordhoff** [ietf-wg-add/draft-ietf-add-ddr#19]
draft-schwartz-svcb-dns specifies that the default alpn parameter is "dot": https://tools.ietf.org/html/draft-schwartz-svcb-dns-02#section-4.1

However, the examples in this draft assume there is no default, e.g.:

```
   _dns.resolver.example.com  7200  IN SVCB 1 . (
        alpn=h2 dohpath=/dns-query{?dns} )
   _dns.resolver.example.com  7200  IN SVCB 2 . (
        alpn=dot )
```

should be something like:

```
   _dns.resolver.example.com  7200  IN SVCB 1 . (
        alpn=h2 dohpath=/dns-query{?dns} no-default-alpn )
   _dns.resolver.example.com  7200  IN SVCB 2 .
```

**Comment by mnordhoff** [comment]
@Habbie points out this will be fixed by bemasc/svcb-dns#2 changing the default, if and when that is merged.

**Comment by Habbie** [comment]
(This draft assuming there is no default is another argument for merging bemasc/svcb-dns#2, I'd say :-) )
